.github/pull_request_template.md

```diff
@@ -9,20 +9,9 @@ and we'll evaluate if we can setup a [CLA](https://en.wikipedia.org/wiki/Contrib
 
 ## For Internal Contributors
```

Comment:
Don't we remove this? because we move the ticket link to the bottom

Reply:
its okay to be here, no? :) I changed title precedence for `## Description Of Changes ` to `### Description Of Changes `